src/MemoryOptions.php

```diff
@@ -33,7 +33,7 @@ class MemoryOptions extends AdapterOptions
      *
      * @link http://php.net/manual/faq.using.php#faq.using.shorthandbytes
      *
-     * @param  string|int $memoryLimit
+     * @param string|int $memoryLimit
```

Comment:
```suggestion
     * @param string|int $memoryLimit String values have to fulfill the php.ini format while the integer value represents the amount of allowed memory in bytes
```

Or something like this, maybe @froschdesign has a better phrase tho.